src/Sylius/Behat/Context/Api/Admin/ManagingOrdersContext.php

```diff
@@ -229,6 +229,66 @@ public function theOrderShouldHavePaymentState(OrderInterface $order, string $pa
         );
     }
 
+    /**
+     * @Then it should have :amount items
+     */
+    public function itShouldHaveAmountOfItems(int $amount): void
+    {
+        Assert::count($this->responseChecker->getValue($this->client->getLastResponse(), 'items'), $amount);
+    }
+
+    /**
+     * @Then the product named :productName should be in the items list
+     */
+    public function theProductShouldBeInTheItemsList(string $productName): void
+    {
+        $items = $this->responseChecker->getValue($this->client->getLastResponse(), 'items');
+
+        foreach ($items as $item) {
+            if ($item['productName'] === $productName) {
+                return;
+            }
+        }
+
+        throw new \InvalidArgumentException('There is no product with given name.');
+    }
+
+    /**
+     * @Then /^the order's shipping total should be ("[^"]+")$/
```

Comment:
I'm afraid it won't work that way in API context because we need to check integer there and the `LexicalContext` won't be called in this turnip notation.

Reply:
Indeed 👍